code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,10 +15,13 @@
 		insert(holder)
 	..()
 
+
 /obj/item/organ/internal/proc/insert(mob/living/carbon/M, special = 0)
 	if(!iscarbon(M) || owner == M)
 		return
 
+	processing_objects -= src
```

Comment:
well right now i need to hook it into for organ rot outside the body...any suggestions on what to do? and damn conflicts..will fix later.